### PR TITLE
DBG: Fix pretty printers if WSL toolchain is used

### PR DESCRIPTION
**Pull request by mchernyavsky**
Fixes https://github.com/intellij-rust/intellij-rust/issues/7197.

changelog: Fix pretty printers if WSL toolchain is used

### CODE REVIEW

File: debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugProcessConfigurationHelper.kt [hunk 1]
```diff
@@ -150,8 +157,14 @@ class RsDebugProcessConfigurationHelper(
         return sysroot
     }
 
-    private fun String.systemDependentAndEscaped(): String =
-        StringUtil.escapeStringCharacters(FileUtil.toSystemDependentName(this))
+    private fun String.systemDependentAndEscaped(): String {
+        val path = if (toolchain is RsRemoteToolchain) {
```
Comment (Undin) [inline debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugProcessConfigurationHelper.kt:161]: Note, I've dropped `RsRemoteToolchain`. Rebase onto master, please

Comment (mchernyavsky) [reply debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugProcessConfigurationHelper.kt]: Done.

**Review by ortem — APPROVED** [review]
LGTM. Now GDB pretty-printers work fine on my machine with WSL.
I found another (probably not related) problem while testing this PR: https://github.com/intellij-rust/intellij-rust/issues/7398

**Review by mchernyavsky — COMMENTED** [review]
bors r=ortem

**Comment by mchernyavsky** [comment]
bors retry

**Comment by Undin** [comment]
bors r=ortem

**Pull request merged**